iosApp/iosApp/Pages/Map/HomeMapViewHandlerExtension.swift

```diff
@@ -220,7 +223,8 @@ extension HomeMapView {
                 stopSequence: stopSequence
             ) : nil,
             routeId: routeId,
-            directionId: vehicle.directionId
+            directionId: vehicle.directionId,
+            routeType: patterns?.routes.first?.type
```

Comment:
Is this ok? Not sure if this is a reliable way to determine the type.

Reply:
My initial thought would be to use the `routeId` that's already in the navigation item to fetch the route from global data where you need it in the trip page, and check the type there. Is there a reason I'm missing that that wouldn't work?

Reply:
While trying to think through that approach, I realized that [this](https://github.com/mbta/mobile_app/pull/408/files#diff-0596f8683751c0aab7db8c7ccfd1736f247c979f0e30eea1e92a8803747f751aR211) is using `trip?.trip.id` as a fallback, which seems wrong, shouldn't it be `trip?.trip.routeId`?

![FEz4HC5XsAMztm3](https://github.com/user-attachments/assets/2f825a40-591e-4ecd-bd93-94f271b4e172 )

Yeah uhh, try changing it and see if it breaks anything, but I think this was my mistake, the previous code before that change was using `trip.routeId`. Then that should make it possible to get the route by ID. And now that I think of it, the trip's routeId should probably be higher priority than the vehicle's, since you've selected a specific trip on a specific route, which may have a vehicle serving a different route.